app/src/main/java/com/ivy/wallet/logic/csv/model/RowMapping.kt

```diff
@@ -35,5 +37,13 @@ data class RowMapping(
 
     val transformTransaction: (Transaction, Category?) -> Transaction = { transaction, _ ->
         transaction
+    },
+
+    val transformAmount: (Double, TransactionType) -> TransactionType = { _ , transactionType ->
```

Comment:
`transformAmount: (Double, TransactionType) -> TransactionType` isn't necessary, `transformTransaction: (Transaction, Category?, csvAmount: Double) -> Transaction` will do just well for that purpose